src/initial-submission/components/FileDetailsForm.tsx

```diff
@@ -42,14 +46,38 @@ const FileDetailsForm = ({ initialValues, schemaFactory, ButtonComponent }: Step
     });
     const schema = schemaFactory(t);
 
+    const validationResolver = useCallback((data: FileDetails) => {
+        try {
+            schema.validateSync(
+                { ...data, manuscriptFile: manuscriptStatus.fileStored.fileName },
```

Comment:
Could write a test for this. Load page with initialValues not having manuscriptFile then try with one and see if it validates correctly?